device-types/Treetech/SDG.yaml

```diff
@@ -0,0 +1,15 @@
+---
+manufacturer: Treetech
+model: SDG
+slug: treetech-sdg
+u_height: 1
+is_full_depth: false
+is_powered: true
+airflow: passive
+comments: Fully aggregate information from any equipment compatible with Modbus and/or DNP3 protocols and redistribute it in a highly customizable way in
+  these protocols and even in those of the IEC 61850 standard.
+power-ports:
```

Comment:
You have created some modules as part of this change, are they to go into this device therefore you need to create the associated `module-bays`